# This file is distributed under the same license as the Django package.
#
# Translators:
# Claude Paroz <claude@2xlibre.net>, 2013-2019
# Claude Paroz <claude@2xlibre.net>, 2013
# Jannis Leidel <jannis@leidel.info>, 2011
# mlorant <maxime.lorant@gmail.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-05-20 02:28+0530\n"
"PO-Revision-Date: 2019-09-18 15:54+0000\n"
"Last-Translator: Claude Paroz <claude@2xlibre.net>\n"
"Language-Team: French (http://www.transifex.com/django/django/language/fr/)\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\admin.py:46
msgid "Personal info"
msgstr "Informations personnelles"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\admin.py:47
msgid "Permissions"
msgstr "Permissions"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\admin.py:50
msgid "Important dates"
msgstr "Dates importantes"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\admin.py:134
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr "L'objet %(name)s avec la clef primaire %(key)r n’existe pas."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\admin.py:144
msgid "Password changed successfully."
msgstr "Mot de passe modifié avec succès"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\admin.py:164
#, python-format
msgid "Change password: %s"
msgstr "Modifier le mot de passe : %s"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\apps.py:15
msgid "Authentication and Authorization"
msgstr "Authentification et autorisation"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\base_user.py:49
msgid "password"
msgstr "mot de passe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\base_user.py:50
msgid "last login"
msgstr "dernière connexion"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:41
msgid "No password set."
msgstr "Aucun mot de passe défini."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:46
msgid "Invalid password format or unknown hashing algorithm."
msgstr ""
"Format de mot de passe non valide ou algorithme de hachage non reconnu."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:88
#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:335
#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:408
msgid "The two password fields didn’t match."
msgstr "Les deux mots de passe ne correspondent pas."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:91
#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:144
#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:180
#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:412
msgid "Password"
msgstr "Mot de passe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:97
msgid "Password confirmation"
msgstr "Confirmation du mot de passe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:100
#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:421
msgid "Enter the same password as before, for verification."
msgstr "Saisissez le même mot de passe que précédemment, pour vérification."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:146
msgid ""
"Raw passwords are not stored, so there is no way to see this user’s "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""
"Les mots de passe ne sont pas enregistrés en clair, ce qui ne permet pas "
"d’afficher le mot de passe de cet utilisateur, mais il est possible de le "
"changer en utilisant <a href=\"{}\">ce formulaire</a>. "

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:187
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""
"Saisissez un %(username)s et un mot de passe valides. Remarquez que chacun "
"de ces champs est sensible à la casse (différenciation des majuscules/"
"minuscules)."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:190
msgid "This account is inactive."
msgstr "Ce compte est inactif."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:253
msgid "Email"
msgstr "Courriel"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:338
msgid "New password"
msgstr "Nouveau mot de passe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:344
msgid "New password confirmation"
msgstr "Confirmation du nouveau mot de passe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:380
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr "Votre ancien mot de passe est incorrect. Veuillez le rectifier."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:383
msgid "Old password"
msgstr "Ancien mot de passe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\forms.py:418
msgid "Password (again)"
msgstr "Mot de passe (à nouveau)"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:265
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:339
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:435
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:495
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:526
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:562
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:598
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:636
msgid "algorithm"
msgstr "algorithme"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:266
msgid "iterations"
msgstr "itérations"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:267
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:345
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:437
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:496
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:527
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:637
msgid "salt"
msgstr "salage"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:268
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:346
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:497
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:528
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:563
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:599
#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:638
msgid "hash"
msgstr "empreinte"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:340
msgid "variety"
msgstr "variété"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:341
msgid "version"
msgstr "version"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:342
msgid "memory cost"
msgstr "coût mémoire"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:343
msgid "time cost"
msgstr "coût temps"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:344
msgid "parallelism"
msgstr "parallélisme"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:436
msgid "work factor"
msgstr "facteur travail"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\hashers.py:438
msgid "checksum"
msgstr "somme de contrôle"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:56
#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:107
msgid "name"
msgstr "nom"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:60
msgid "content type"
msgstr "type de contenu"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:62
msgid "codename"
msgstr "nom de code"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:67
msgid "permission"
msgstr "permission"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:68
#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:110
msgid "permissions"
msgstr "permissions"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:117
msgid "group"
msgstr "groupe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:118
#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:241
msgid "groups"
msgstr "groupes"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:232
msgid "superuser status"
msgstr "statut super-utilisateur"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:235
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr ""
"Précise que l’utilisateur possède toutes les permissions sans les assigner "
"explicitement."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:244
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""
"Les groupes dont fait partie cet utilisateur. Celui-ci obtient tous les "
"droits de tous les groupes auxquels il appartient."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:252
msgid "user permissions"
msgstr "permissions de l’utilisateur"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:254
msgid "Specific permissions for this user."
msgstr "Permissions spécifiques à cet utilisateur."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:325
msgid "username"
msgstr "nom d’utilisateur"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:328
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""
"Requis. 150 caractères maximum. Uniquement des lettres, nombres et les "
"caractères « @ », « . », « + », « - » et « _ »."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:331
msgid "A user with that username already exists."
msgstr "Un utilisateur avec ce nom existe déjà."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:334
msgid "first name"
msgstr "prénom"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:335
msgid "last name"
msgstr "nom"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:336
msgid "email address"
msgstr "adresse électronique"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:338
msgid "staff status"
msgstr "statut équipe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:340
msgid "Designates whether the user can log into this admin site."
msgstr "Précise si l’utilisateur peut se connecter à ce site d'administration."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:343
msgid "active"
msgstr "actif"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:346
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""
"Précise si l’utilisateur doit être considéré comme actif. Décochez ceci "
"plutôt que de supprimer le compte."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:350
msgid "date joined"
msgstr "date d’inscription"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:359
msgid "user"
msgstr "utilisateur"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\models.py:360
msgid "users"
msgstr "utilisateurs"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:102
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] ""
"Ce mot de passe est trop court. Il doit contenir au minimum %(min_length)d "
"caractère."
msgstr[1] ""
"Ce mot de passe est trop court. Il doit contenir au minimum %(min_length)d "
"caractères."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:112
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] ""
"Votre mot de passe doit contenir au minimum %(min_length)d caractère."
msgstr[1] ""
"Votre mot de passe doit contenir au minimum %(min_length)d caractères."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:151
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr "Le mot de passe est trop semblable au champ « %(verbose_name)s »."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:157
msgid "Your password can’t be too similar to your other personal information."
msgstr ""
"Votre mot de passe ne peut pas trop ressembler à vos autres informations "
"personnelles."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:183
msgid "This password is too common."
msgstr "Ce mot de passe est trop courant."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:188
msgid "Your password can’t be a commonly used password."
msgstr ""
"Votre mot de passe ne peut pas être un mot de passe couramment utilisé."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:198
msgid "This password is entirely numeric."
msgstr "Ce mot de passe est entièrement numérique."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\password_validation.py:203
msgid "Your password can’t be entirely numeric."
msgstr "Votre mot de passe ne peut pas être entièrement numérique."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\templates\registration\password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr "Réinitialisation du mot de passe sur %(site_name)s"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\validators.py:12
msgid ""
"Enter a valid username. This value may contain only English letters, "
"numbers, and @/./+/-/_ characters."
msgstr ""
"Saisissez un nom d’utilisateur valide. Il ne peut contenir que des lettres "
"non accentuées, des nombres ou les caractères « @ », « . », « + », « - » et "
"« _ »."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""
"Saisissez un nom d’utilisateur valide. Il ne peut contenir que des lettres, "
"des nombres ou les caractères « @ », « . », « + », « - » et « _ »."

#: .\env\lib\python3.6\site-packages\django\contrib\auth\views.py:160
msgid "Logged out"
msgstr "Déconnecté"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\views.py:217
msgid "Password reset"
msgstr "Réinitialisation du mot de passe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\views.py:244
msgid "Password reset sent"
msgstr "Message de réinitialisation du mot de passe envoyé"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\views.py:254
msgid "Enter new password"
msgstr "Saisissez un nouveau mot de passe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\views.py:314
msgid "Password reset unsuccessful"
msgstr "Échec lors de la mise à jour du mot de passe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\views.py:322
msgid "Password reset complete"
msgstr "Mise à jour du mot de passe terminée"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\views.py:334
msgid "Password change"
msgstr "Modification du mot de passe"

#: .\env\lib\python3.6\site-packages\django\contrib\auth\views.py:357
msgid "Password change successful"
msgstr "Mot de passe modifié avec succès"
